### PR TITLE
Mention .spacemacs.d in documentation

### PR DESCRIPTION
**Pull request by justbur**

### CODE REVIEW

**Comment by justbur** [comment]
See #2760 for why indentation is changing in toc

File: doc/DOCUMENTATION.org [hunk 1]
```diff
@@ -442,6 +443,20 @@ manually from the template file =~/.emacs.d/core/templates/.spacemacs.template=
     $ cp ~/.emacs.d/core/templates/.spacemacs.template ~/.spacemacs
 #+end_src
 
+*** Alternative setup
+Since v0.104 you have the option of using =~/.spacemacs.d/init.el= for your
+dotfile instead of =~/.spacemacs=. If you want to use this option, simply move
+=~/.spacemacs= to =~/.spacemacs.d/init.el=. =~/.spacemacs= will alway takes
```
Comment (MerelyAPseudonym) [inline doc/DOCUMENTATION.org]: a couple typos at the end of this line—it should probably be: `=~/.spacemacs= will always take`

Comment (justbur) [inline doc/DOCUMENTATION.org]: yes thank you.

**Comment by syl20bnr** [comment]
Thank you again :-)
Cherry-picked into develop branch, you can safely delete your branch.

**Pull request closed without merging**